### PR TITLE
[IR-3423] Unit Test - Spatial Physics coverage to 80%

### PR DESCRIPTION
**Pull request by heysokam**
## References
https://tsu.atlassian.net/browse/IR-3423

### CODE REVIEW

File: packages/spatial/src/physics/systems/PhysicsPreTransformSystem.test.ts [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+  function assertNotDirty(entity: Entity, id: number = 0): void {
+    if (!id) {
+      assert.equal(TransformComponent.dirtyTransforms[entity], false)
+      return
+    }
+    if (TransformComponent.dirtyTransforms[entity] === undefined) return
+    assert.equal(TransformComponent.dirtyTransforms[entity], false)
+  }
+
+  describe('setDirty', () => {
```
Comment (HexaField) [inline packages/spatial/src/physics/systems/PhysicsPreTransformSystem.test.ts]: A setDirty test is unnecessary, as the function definition exists only to minimize garbage. We only need to test functionality unique to the module.

File: packages/spatial/src/physics/systems/PhysicsPreTransformSystem.ts [hunk 2]
```diff
@@ -195,3 +195,11 @@ export const PhysicsPreTransformSystem = defineSystem({
   insert: { after: TransformDirtyUpdateSystem },
   execute
 })
+
+export const _PhysicsPreTransformFunctions = {
```
Comment (HexaField) [inline packages/spatial/src/physics/systems/PhysicsPreTransformSystem.ts]: Let's remove the underscore from this definition name. Even though these functions are unlikely to be used externally, we should keep the namespace clear.

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/spatial/src/physics/systems/PhysicsPreTransformSystem.ts (hunk 2 above)
Comment (heysokam) [reply packages/spatial/src/physics/systems/PhysicsPreTransformSystem.ts]: Fixed with 89c3e14

File: packages/spatial/src/physics/systems/PhysicsPreTransformSystem.test.ts (hunk 1 above)
Comment (heysokam) [reply packages/spatial/src/physics/systems/PhysicsPreTransformSystem.test.ts]: Fixed with 89c3e14

**Review by HexaField — APPROVED** [review]

**Pull request merged**